helm-chart/Chart.yaml

```diff
@@ -14,10 +14,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.2
+version: 0.1.3
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "3.14.0"
+appVersion: "7.0.1"
```

Comment:
Could you please explain why this version bump and not 3.15.0?

Reply:
Oh, sorry, This was a copy-paste from a local copy of the chart.

Reply:
So one issue with this is that the helm chart could support more than one version of the core since there are core changes that don't affect any of the configs provided to it. In that case, updating the helm chart version is unnecessary. What do you think is the right solution in this case?

Reply:
The helm chart has two "versions", the version of chart itself and the version of application.
The general rule is:
- chart `version` is updated whenever there are changes in some chart (e.g. like this one with adding new variable)
- chart `appVersion` is changed to indicate what version of app is bundled.
(The comments in helm chart itself describe it in details)


Ideally how the flow should look like:
1. You update something in supertokens-core and publish new version `x.y.z`
2. New docker image is built with tag `x.y.z`
3. In helm chart only `appVersion` is updated to `x.y.z.` and in values the proper docker image tag is set.

However in this case if you have for example docker image with tag `7.0` which will contain all `patch` fixes, we can just put something like `7.0.0` into `appVersion` in order to show users which "major" version is bundled in this chart.

Reply:
Gotcha! We are planning on making the docker tag x.y.z as well, but until then, what you said about making it 7.0.0 makes sense. Two more questions:
- I noticed the `tag` in `helm-chart/values.yaml` was also changed to `7.0.1`. What does `tag` mean?
- Setting the `appVersion` seems to have nothing to do with the actual docker image being pulled in right? So in this case, it will always pull in the `latest` image? If so, maybe we should set the image tag to `7.0` as well, since auto pulling `latest` is a bad idea (since it can auto pull in breaking changes)

Reply:
The tag was changed in #27. https://github.com/supertokens/supertokens-docker-postgresql/pull/27/files#diff-15210b6a69967e9488632bf12df699fc922b886f3b38d79fdef46502d8bdbd30R5

And tag is just a variable, that's later used in `Deployment` and basically represents which docker image tag will be pulled.

Reply:
Yea, that's odd. The tag should be changed to `7.0`. Which i see that you have done. Thanks!